### PR TITLE
Adds iron pills to NanoMeds

### PR DESCRIPTION
**Pull request by ghost**
So many people don't know that iron in game actually does regenerate blood

:cl: More Robust Than You
add: Nanomeds now have iron pills. YES IRON IS USEFUL.
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -71,84 +71,91 @@
...
 	icon_state = "pill18"
 	list_reagents = list("insulin" = 50)
-	roundstart = 1
+	roundstart = TRUE
+
+/obj/item/reagent_containers/pill/iron
+	name = "iron pill"
+	desc = "Handles blood loss."
+	icon_state = "pill6"
+	list_reagents = list("iron" = 50)
```
Comment (lordpidey) [inline code/modules/reagents/reagent_containers/pill.dm]: How much blood does 50 iron regenerate, assuming normal metabolization?

**Comment by swindly** [comment]
But that's what the saline-glucose solution bottles are for.

**Comment by MoreRobustThanYou** [comment]
@swindly IIRC, saline-glucose is a blood substitute (i.e put it in an iv drip), not a blood regenerator

**Comment by QualityVan** [comment]
I really don't see why this is necessary.

**Comment by SgtxRootbeer** [comment]
@MoreRobustThanYou saline-glucose can be OD'd unlike iron, meaning its a short-term bandaid

File: code/modules/reagents/reagent_containers/pill.dm (hunk 1 above)
Comment (RandomMarine) [reply code/modules/reagents/reagent_containers/pill.dm]: 62.5 blood after a little over four minutes.

**Comment by jughu** [comment]
Uneeded theres litteraly a chemistry machine that mass produces iron in the very same room ask the chemists to make some iron pills litteraly take seconds

**Comment by MoreRobustThanYou** [comment]
@jughu the chemists are usually either retarded, on fire, or busy making meth

**Comment by Bawhoppen** [comment]
Literally powercreep

**Comment by Cyberboss** [comment]
@MoreRobustThanYou Sure, I'm making meth, but I can't even remember the last time anyone asked me to make ANYTHING >:(

**Comment by MoreRobustThanYou** [comment]
@Cyberboss you mean assistants don't show up at medbay screeching and demanding you make [insert chem here]?

**Comment by Cyberboss** [comment]
Actually no

**Comment by ShiggyDiggyDo** [comment]
I don't really think it's necessary, it's literally a one chemical pill that the chemist can make in seconds. If the chemists are too retarded to click one button just ask the CMO or the AI to let you into chemistry.

**Comment by ShizCalev** [comment]
Unneeded. Just ask the Chemists right next to the NanoMeds to give you some if you absolutely need it, no need to remove player-player interaction and have the best meds in a vendor at roundstart.

**Pull request closed without merging**